.travis.yml

```diff
@@ -1,14 +0,0 @@
-language: csharp
-mono: none
-dotnet: 2.1.2
-cache:
-  directories:
-  - $HOME/.nuget/packages
-before_install:
-  - eval "$(curl -Ss https://raw.githubusercontent.com/neovim/bot-ci/master/scripts/travis-setup.sh) nightly-x64"
-script:
-  - dotnet run --project
-    src/NvimClient.APIGenerator/NvimClient.APIGenerator.csproj
-    src/NvimClient.API/NvimAPI.generated.cs
```

Comment:
the old script ran the generator, do we (plan to) have that in the GHA workflow?

even if it doesn't auto-create a PR or something, it's useful to exercise it I think?

Reply:
I wondered about this; my thought for leaving it out is this way the file in the published package is the same as the one in the repo, so there aren't any unexpected discrepancies between the two.

Then we can just add another workflow that does update the API and create a PR if we want to automate that.